### DISCUSSION TITLE
What version dependency of LibMediaProvider is needed?

### DISCUSSION THREAD

**Discussion by F3LX-dev** [DakJaniels/LuiExtended#301] [Q&A] [answered]
I have LibMediaProvider-1.0 r31 installed which looks like the latest version on the esoui page but in-game it want a newer version?

**Comment by DakJaniels** [comment]
Very odd. Try reinstalling from minion or installing manually from esoui. https://www.esoui.com/downloads/info56-LibMediaProvider.html

**Reply by F3LX-dev** [reply]
the one from esoui counts as outdated
![image](https://github.com/user-attachments/assets/8673455b-7b6d-49b7-a898-af4a2f0e820c)

**Comment by F3LX-dev** [comment, marked as answer]
Works now following a recent update
